go-controller/pkg/cni/bandwidth_test.go

```diff
@@ -0,0 +1,280 @@
+				{"CombinedOutput", []string{}, []interface{}{nil, nil}},
+			},
+			runnerInstance: mockKexecIface,
+		},
+	}
+	for i, tc := range tests {
+		t.Run(fmt.Sprintf("%d:%s", i, tc.desc), func(t *testing.T) {
+			if tc.onRetArgsKexecIface != nil {
+				for _, item := range tc.onRetArgsKexecIface {
+					ifaceCall := mockKexecIface.On(item.OnCallMethodName)
+					for _, arg := range item.OnCallMethodArgType {
+						ifaceCall.Arguments = append(ifaceCall.Arguments, mock.AnythingOfType(arg))
+					}
+					for _, ret := range item.RetArgList {
+						ifaceCall.ReturnArguments = append(ifaceCall.ReturnArguments, ret)
+					}
+					ifaceCall.Once()
+				}
+			}
+			if tc.onRetArgsCmdList != nil {
+				for _, item := range tc.onRetArgsCmdList {
+					mockCall := mockCmd.On(item.OnCallMethodName)
+					for _, arg := range item.OnCallMethodArgType {
+						mockCall.Arguments = append(mockCall.Arguments, mock.AnythingOfType(arg))
+					}
+					for _, ret := range item.RetArgList {
+						mockCall.ReturnArguments = append(mockCall.ReturnArguments, ret)
+					}
+					mockCall.Once()
+				}
+			}
+			runner = tc.runnerInstance
+
+			e := clearPodBandwidth("sandboxID")
+
+			if tc.expectedErr != nil {
+				assert.Error(t, e)
+			}
+
+			mockCmd.AssertExpectations(t)
+			mockKexecIface.AssertExpectations(t)
+		})
+	}
+}
+
+func TestSetPodBandwidth(t *testing.T) {
+	mockKexecIface := new(mock_k8s_io_utils_exec.Interface)
+	mockCmd := new(mock_k8s_io_utils_exec.Cmd)
+
+	tests := []struct {
+		desc                string
+		expectedErr         error
+		onRetArgsKexecIface []ovntest.TestifyMockHelper
+		onRetArgsCmdList    []ovntest.TestifyMockHelper
+		runnerInstance      kexec.Interface
+		egressBPS           int64
+	}{
+		{
+			desc:        "Test code path when both ingressBPS is greater than zero and ovsCreate returns an error",
+			expectedErr: nil,
+			onRetArgsKexecIface: []ovntest.TestifyMockHelper{
+				{"Command", []string{"string", "string", "string", "string", "string", "string", "string", "string", "string"}, []interface{}{mockCmd}}},
+			onRetArgsCmdList: []ovntest.TestifyMockHelper{
+				{"CombinedOutput", []string{}, []interface{}{nil, fmt.Errorf("failed to run ovsCreate")}}},
+			runnerInstance: mockKexecIface,
+			egressBPS:      0,
+		},
+		{
+			desc:        "Test code path when inressBPS is greater than zero and ovsSet returns an error",
+			expectedErr: nil,
+			onRetArgsKexecIface: []ovntest.TestifyMockHelper{
+				{"Command", []string{"string", "string", "string", "string", "string", "string", "string", "string", "string"}, []interface{}{mockCmd}},
+				{"Command", []string{"string", "string", "string", "string", "string", "string", "string", "string", "string"}, []interface{}{mockCmd}},
+			},
+			onRetArgsCmdList: []ovntest.TestifyMockHelper{
+				{"CombinedOutput", []string{}, []interface{}{nil, nil}},
+				{"CombinedOutput", []string{}, []interface{}{nil, fmt.Errorf("Failed to run ovsSet")}},
```

Comment:
you will get linter issues if the error message starts with a capital letter, hence your CI test failure

Reply:
Thank you, will change that